docs/use/utilities/visualizer.md

```diff
@@ -1,34 +1,41 @@
 ---
-title: Visualizer
-description: See the IOTA Tangle in real time confirming messages and building the unique IOTA Directed Acyclic Graph.
+title: Network Visualizer
+description: See the IOTA Tangle in real-time confirming messages and building the unique IOTA Directed Acyclic Graph.
 ---
 
 
-# Heading 1 (Page Title)
+# Network Visualizer
 
-content - introduction to the topic
+Network Visualizers show you the activity of messages happening in an IOTA Tangle in real-time. They are a powerful tool for the researchers to help analyzing the complex structure and also produce often impressive and beautiful impressions of the so complex technology in IOTA. Many people find it fascinating to observe the Graph of messages grow and change under different network conditions.
 
+IOTA provides visualizers in the Network explorer Websites for the following networks:
 
-## Heading 2 (Sub topic 1)
 
-content 
+## Mainnet
 
-## Heading 2 (Sub topic 2)
+The currently live and used Mainnet of IOTA. Implemented Protocol stage is Chrysalis (IOTA 1.5). This is the network where all real world-data and value transfers happen:
 
-content
+- **[Mainnet Visualizer](https://explorer.iota.org/mainnet/visualizer/)**
 
 
+## Testnet
 
-<br/>
+Testnet used for further development of the current IOTA 1.5 Chrysalis Protocol. All transactions here use Testnet tokens without value. It is used by the community and the IOTA Foundation to test and develop applications and new features
 
-----
+- **[Testnet Visualizer](https://explorer.iota.org/testnet/visualizer/)**
 
-<br/>
 
-**need to add a footer section for further content - (three cards)**
+## Legacy Mainnet
+
+This is the still active, but old Mainnet of the IOTA 1.0 Protocol implementation. The only transactions happening here are migrations of the tokens into the new IOTA 1.5 Maninnet of users that did not yet migrate their Tokens into the new network.
+
+- **[Legacy Mainnet Visualizer](https://explorer.iota.org/legacy-mainnet/visualizer/)**
+
+## IOTA 2.0 Decentralized DevNet
+
+The implementation of the upcoming fully decentralized IOTA 2.0 Protocol. The network here does not use or need a coordinator anymore. Used with Testnet tokens to fine-tune, test and optimize the upcoming development stage of the IOTA Protocol together with the IOTA community.
+
+- **[IOTA 2.0 DevNet](https://v2.iota.org/visualizer)**
+
 
```

Comment:
When comnet :(

Reply:
hmmm - does nobody have one running? If so, we should implement that...